### PR TITLE
Combine features and components

### PR DESCRIPTION
**Pull request by sequencemedialimited**
- Acceptance tests are in `features`
- Component acceptance tests are in `components`

I've moved any shell commands from `Makefile` into shell scripts in `scripts`

Most of the "files changed" reflect files _added_ to the project from `fb-component-acceptance-tests`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,37 +25,37 @@ setup: .runner .features .submitter .datastore .filestore .pdf-generator .servic
...
 	docker-compose build --parallel
 
 serve: build
 	docker-compose up -d
+	./scripts/wait_for_datastore_app.sh
+	./scripts/wait_for_filestore_app.sh
+	./scripts/wait_for_submitter_app.sh
+	./scripts/wait_for_service_token_cache_app.sh
 	./scripts/wait_for_features_apps.sh
+	./scripts/wait_for_components_apps.sh
```
Comment (brenetic) [inline Makefile]: Was outputting the SHA's a useful thing?

Comment (sequencemedialimited) [reply Makefile]: That's a good question. It was implemented by Emile back in October/November

All it prints is `{"APP_SHA":"HEAD"}` so ... suggestions? I mean, it _confirms_ that the response is as expected, but thassit

(Would it error if grep doesn't find that ..?)

**Comment by sequencemedialimited** [comment]
> FFTMWA

WUT

**Comment by brenetic** [comment]
Feel Free To Merge With Abandon

**Comment by sequencemedialimited** [comment]
Google's top result was "Your Guess Is As Good As Mine"

**Pull request merged**